samples/lcd/lcd-more-sections/main.cpp

```diff
@@ -0,0 +1,88 @@
+/**
+* Sample code for display data on an LCD with more than the default sections
+*/
+
+#include <EVT/dev/LCD.hpp>
+#include <EVT/dev/LED.hpp>
+#include <EVT/io/UART.hpp>
+#include <EVT/manager.hpp>
+#include <EVT/utils/time.hpp>
+#include <cstdio>
+
+namespace DEV = EVT::core::DEV;
+namespace IO = EVT::core::IO;
+namespace time = EVT::core::time;
+
+constexpr uint32_t SPI_SPEED = SPI_SPEED_500KHZ;
+
+constexpr uint8_t deviceCount = 1;
+
+IO::GPIO* devices[deviceCount];
+
+int main() {
+    // Initialize system
+    EVT::core::platform::init();
+
+    // Setup UART
+    IO::UART& uart = IO::getUART<IO::Pin::UART_TX, IO::Pin::UART_RX>(9600);
+
+    // Uses HUDL 1.0 Pins
+    IO::GPIO& regSelect = IO::getGPIO<IO::Pin::PA_3>(EVT::core::IO::GPIO::Direction::OUTPUT);
+    IO::GPIO& reset = IO::getGPIO<IO::Pin::PB_3>(EVT::core::IO::GPIO::Direction::OUTPUT);
+    devices[0] = &IO::getGPIO<IO::Pin::PB_12>(EVT::core::IO::GPIO::Direction::OUTPUT);
+    devices[0]->writePin(IO::GPIO::State::HIGH);
+
+    // Setup SPI
+    IO::SPI& spi = IO::getSPI<IO::Pin::SPI_SCK, IO::Pin::SPI_MOSI>(devices, deviceCount);
+    spi.configureSPI(SPI_SPEED, SPI_MODE0, SPI_MSB_FIRST);
+
+    // Sets up LCD
+    uart.printf("Creating LCD Object...\n\r");
+    EVT::core::DEV::LCD lcd(regSelect, reset, spi, 12, 3, 2);
+    uart.printf("Initializing LCD...\n\r");
+    lcd.initLCD();
+    lcd.clearLCD();
+
+    // The segment titles
+    char* titles[12] = {
+        "B Voltage",
+        "Speed",
+        "RPM",
+        "Temp 1",
+        "Temp 2",
+        "Temp 3",
+        "Status 1",
+        "PVC Stat",
+        "Torque",
+        "Extra Data",
+        "Extra Data",
+        "Extra Data"};
+    lcd.setDefaultSections(titles);
+
+    // Set the default
+    lcd.displaySectionHeaders();
+    lcd.setTextForSection(0, "3.2 v");
+    lcd.setTextForSection(2, "25 MPH");
```

Comment:
```suggestion
    lcd.setTextForSection(1, "25 MPH");
```
Should this be changed?